internal/ingress/controller/parser/util/objectmeta.go

```diff
@@ -0,0 +1,28 @@
+package util
+
+import (
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+// K8sObjectInfo describes a Kubernetes object.
+type K8sObjectInfo struct {
+	Name        string
+	Namespace   string
+	Annotations map[string]string
+}
```

Comment:
Question (not requesting changes): Why not use metav1.ObjectMeta at this point?

Reply:
Discussed out of band.

As per PR description: 

> I considered using k8s ObjectMeta instead of implementing K8sObjectInfo but that would violate separation between Kong and Kubernetes data formats that I wanted to achieve in package kongstate.